README.md

```diff
@@ -62,6 +62,9 @@ defmodule Decoding.Pipeline do
 end
 ```
 
+[Membrane.H264.Parser](https://github.com/membraneframework/membrane_h264_plugin) is our actively maintained, pure Elixir implementation of the h264 parser. 
+We recommend using it instead of `H264.FFmpeg.Parser`.
```

Comment:
I'm not sure about recommending it yet - if we want to recommend it, I think we should go all in and deprecate the old parser, include it in the example and probably make the pure-elixir plugin a dependency of this one.

It seems we're not far from the feature parity, but not there yet.

Thus, for now I'd make the suggestion here softer:

```suggestion
`Membrane.H264.FFmpeg.Parser` is not actively developed and will soon be deprecated in favour of our pure Elixir implementation of the H264 parser. We encourage you to try out [`Membrane.H264.Parser`](https://hexdocs.pm/membrane_h264_plugin/Membrane.H264.Parser.html) from [membrane_h264_plugin](https://github.com/membraneframework/membrane_h264_plugin).
```